docs/readthedocs/source/doc/PythonAPI/LLM/optimize.rst

```diff
@@ -1,10 +1,24 @@
 BigDL-LLM PyTorch API
 =====================
 
-optimize model
+Optimize Model
 ----------------------------------------
 
-.. automodule:: bigdl.llm.optimize
+You can run any PyTorch model with one-line ``optimize_model`` to optimize, regardless of the library or API you are using.
+
+.. automodule:: bigdl.llm
     :members: optimize_model
     :undoc-members:
     :show-inheritance:
+
+
+
+Load Optimized Model
+----------------------------------------
+
+To save model space and speedup loading processes, you can save the model after low-bit optimization and load the saved low-bit model. Saving and loading operations can across various machines, regardless of their operating systems.
```

Comment:
can work across

Reply:
To save model space and speedup loading processes... =>
Maybe it is more clear to be

To avoid high resource consumption during the loading processes of the original model, we provide save/load API to support the saving of model after low-bit optimization and the loading of the saved low-bit model.

Saving and loading operations can across various machines => Saving and loading operations are platform-independent, ...